libs/shared-shadcn-ui-components/src/components/ui/button.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react"
+import { Slot } from "@radix-ui/react-slot"
+import { cva, type VariantProps } from "class-variance-authority"
+
+import { cn } from "@talent-connect/shared-utils"
+
+const buttonVariants = cva(
+  "tw-inline-flex tw-items-center tw-justify-center tw-whitespace-nowrap tw-rounded-md tw-text-sm tw-font-medium tw-transition-colors focus-visible:tw-outline-none focus-visible:tw-ring-1 focus-visible:tw-ring-ring disabled:tw-pointer-events-none disabled:tw-opacity-50",
+  {
+    variants: {
+      variant: {
+        default:
+          "tw-bg-primary tw-text-primary-foreground tw-shadow hover:tw-bg-primary/90",
+        destructive:
+          "tw-bg-destructive tw-text-destructive-foreground tw-shadow-sm hover:tw-bg-destructive/90",
+        outline:
+          "tw-border tw-border-input tw-bg-background tw-shadow-sm hover:tw-bg-accent hover:tw-text-accent-foreground",
+        secondary:
+          "tw-bg-secondary tw-text-secondary-foreground tw-shadow-sm hover:tw-bg-secondary/80",
+        ghost: "hover:tw-bg-accent hover:tw-text-accent-foreground",
+        link: "tw-text-primary tw-underline-offset-4 hover:tw-underline",
+      },
+      size: {
+        default: "tw-h-9 tw-px-4 tw-py-2",
+        sm: "tw-h-8 tw-rounded-md tw-px-3 tw-text-xs",
+        lg: "tw-h-10 tw-rounded-md tw-px-8",
+        icon: "tw-h-9 tw-w-9",
+      },
+    },
+    defaultVariants: {
+      variant: "default",
+      size: "default",
+    },
+  }
+)
```

Comment:
Why did we decide to use `tw` prefix? Is it again a convetion from shadcn? I feel like this is introducing a project specific convention where it can confuse beginner volunteers. What are the benefits of using this?

Reply:
We decided to use `tw-` prefix because we are layering Tailwind classes on top of our existing custom CSS classes. When I first integrated TailwindCSS without using `tw-` prefix, I saw some styles broken on both platforms because of the naming conflicts. Also, `tw-` prefix lets us easily distinguish Tailwind classes from the rest.